GoogleForCreators/web-stories-wp
# UI Simplification: Colors

## Summary
Moves the Saved colors panel functionality to the Color Picker.


## Relevant Technical Choices



## To-do

- [x] Make the confirmation dialogue work



## User-facing changes
| Before | After |
|-------|------|
| <img width="259" alt="Screenshot 2021-11-04 at 11 42 36" src="https://user-images.githubusercontent.com/3294597/140291962-f0ab35ad-d490-4c08-9df7-a9138efa3aaa.png"> | <img width="258" alt="Screenshot 2021-11-04 at 11 34 34" src="https://user-images.githubusercontent.com/3294597/140291531-abbf092b-435f-4941-b64f-5608b51214d7.png"> |
| <img width="298" alt="Screenshot 2021-11-04 at 11 43 37" src="https://user-images.githubusercontent.com/3294597/140292127-776e6c41-d37c-4ba4-9327-7e8eb9debe1f.png"> | <img width="301" alt="Screenshot 2021-11-04 at 11 39 15" src="https://user-images.githubusercontent.com/3294597/140291540-a9b25ca8-2b37-470d-aaec-d08cd0c1299e.png"> |
| <img width="259" alt="Screenshot 2021-11-04 at 11 44 45" src="https://user-images.githubusercontent.com/3294597/140292257-919ee736-53bf-4e12-9e29-e7d5482d7d5e.png"> | <img width="263" alt="Screenshot 2021-11-04 at 11 38 17" src="https://user-images.githubusercontent.com/3294597/140291542-d7e69a80-8e73-4894-a52d-02a3ae695ef3.png"> |
| <img width="300" alt="Screenshot 2021-11-04 at 11 45 34" src="https://user-images.githubusercontent.com/3294597/140292385-dcd97de3-45a0-4cf3-8737-e91da6c2d0e9.png"> | <img width="260" alt="Screenshot 2021-11-04 at 11 37 55" src="https://user-images.githubusercontent.com/3294597/140291545-fa2ad268-9082-4286-abd5-e75081234bf9.png"> |




## Testing Instructions






This PR can be tested by following these steps:

1. Add a shape element and very that the "Style" panel is now called the "Color" panel instead.
2. Verify there is no Saved colors panel anymore.
3. Open the Color picker and verify the Basic color picker view now includes "Current story", "Saved Story" sections.
4. Verify that new colors can be added from the Color Picker
5. Verify that the edit mode of the Color Picker works as expected and allows removing colors. Note that "Default colors" can not be deleted.
6. Verify that when clicking on a Default color while in Edit mode, the Edit mode exits and the color is applied to the element.
7. Verify the Eyedropper (Sample color) in the Basic view works as expected.
8. Go to the Custom view of the color picker, verify the button "Add" now says "Add to Saved Colors"


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #9464
